### ISSUE TITLE
Feature Request: Support es6 modules

### ISSUE THREAD

**Issue by backspaces** [paperjs/paper.js#1481]
# Description/Steps to reproduce

There appears to be no way to import paper as an es6 module.

Using https://unpkg.com/paper/, and https://unpkg.com/paper?module, I can find no modules.

# Link to reproduction test-case

https://unpkg.com/paper?module fails. Now this could be an issue with package.json not having a module field.

# Expected result

The ability to use 'import paper from 'https://unpkg.com/paper?module'

# Additional information

This is browser independent, other than being "modern", i.e. supporting es6 modules.

The traditional solution for upgrading is to:
- Convert the source tree to individual es6 modules
- Use Rollup to create two bundles: UMD for browser/node and ESM for es6 modules.

**Comment by wgottschalk** [comment]
I agree with this. I think it's really important to be able to use in modern applications. It could allow us to build declarative components with all the modern vdom frameworks

**Comment by renderpci** [comment]
I agree. 
I think that will be a great improvement to Paper, but I would only bundle ESM format and would dismiss / discard CommonJS format, node will remove it support in the future versions. In other words; would center the work in ESM only.

**Comment by backspaces** [comment]
The key is that Paper does not support vanilla es6 modules. Your example is
within a framework like webpack.

The confusion arises due to the workflow and frameworks many projects use:
webpack, babel, React and so on.

These were needed due to the perceived complexity of Javascript and es6.
especially the lack of support for "bare modules" and "import maps".

Fortunately, skypack, jspm, unpkg and other modern CDNs actually support
creating modules for you, even with bare imports. In terms of local npm
based development, snowpack completely solves the problem.

**Comment by toddwong** [comment]
Both Edge and Chrome have supported modules and import maps. A es module could ease the developing/debugging procudure.

**Comment by ruben-podadera** [comment]
+1

**Comment by pearcetm** [comment]
I'd like to add another vote for making this fantastic library compatible with vanilla es6 modules

**Comment by ea42gh** [comment]
I'd like to add my vote!
I amnaged to make it work in jupyter lab with pyviz panel,
but had to load paper.js in an explicit script tag:

[https://discourse.holoviz.org/t/geogebra-substitutes-paper-js/8489](https://discourse.holoviz.org/t/geogebra-substitutes-paper-js/8489)

**Comment by MarcSkovMadsen** [comment]
You might try this import `import paper from 'https://esm.sh/paper'` @ea42gh .

**Comment by bleeptrack** [comment]
> You might try this import `import paper from 'https://esm.sh/paper'` [@ea42gh](https://github.com/ea42gh) .

I'm in the process of finding out how to use paper.js with esm.sh. Did you get this working? I'm running into: setting getter-only property "document".
I'm using an import map like:
`"paper-full": "https://esm.sh/paper"`
